deployments/ansible/roles/collector/tasks/otel_win_install.yml

```diff
@@ -77,21 +156,18 @@
 - name: Merge custom config into the default config
   ansible.builtin.import_tasks: config_override.yml
   when:
-    - splunk_collector_msi_is_configurable is undefined
```

Comment:
MSI installation cannot directly set custom variables. This means that the old functionality for setting service registry keys must still be used for custom variables, even if the MSI is configurable.